File: Niveau 1/8 – Répétitions conditionnées/5 - Département de chimie mélange explosif.py
nbMesureEnvisage = int(input())
minTemp = int(input())
maxTemp = int(input())

for i in range(nbMesureEnvisage):
    if minTemp <= int(input()) <= maxTemp:
        print("Rien à signaler")
    else:
        print("Alerte !!")
        break

File: Niveau 1/8 – Répétitions conditionnées/5 - Département de chimie mélange explosif.cpp
#include <iostream>
using namespace std;
int main()
{
  int nbTotalMesure, tempMin, tempMax;
  cin >> nbTotalMesure >> tempMin >> tempMax;
  for (int i = 0; i < nbTotalMesure; i++)
  {
    int temp;
    cin >> temp;
    if (temp >= tempMin && temp <= tempMax)
    {
      cout << "Rien à signaler" << endl;
    }
    else
    {
      cout << "Alerte !!" << endl;
      break;
    }
  }
}